dictionaryutils/__init__.py

```diff
@@ -239,3 +239,19 @@ def resolve_schema(self, obj, root):
             return [self.resolve_schema(item, root) for item in obj]
         else:
             return obj
+
+    def allow_nulls(self):
+        for key, value in self.schema.items():
+            try:
+                try:
+                    required = value["required"]
+                except KeyError:
+                    required = []
+                for k, v in value["properties"].items():
+                    if k not in required and "type" in v:
+                        if "null" not in v["type"]:
+                            if type(v["type"]) != list:
```

Comment:
`if not isinstance(x, list)` is the recommended pythonic way of checking if something is a list